FULL_HELP_DOCS.md

```diff
@@ -1073,6 +1073,7 @@ By default, when starting a testnet container, without any optional arguments, i
 
 ###### **Options:**
 
+* `-c`, `--container-name <CONTAINER_NAME>` — Optional argument to specify the container name
```

Comment:
I think we could drop the word container, because most of the options could be prefixed with it or not and mean the same thing. We're also unlikely to have a separate name and container name, so simply name is sufficient.

In the interest of preserving short options for options that become most meaningful I suggest not adding short options when options are first added, and instead add them later when folks get irritated by the long ones.

```suggestion
* `--name <NAME>` — Optional argument to specify the container name
```

Reply:
Cool, that all makes sense.